### PR TITLE
Edit openssl cnf

### PR DESCRIPTION
**Pull request by thefrieddan1**
to handle UnsafeLegacyRenegotiation

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
Related: https://jira-hq.paloaltonetworks.local/browse/CIAC-4962

## Description
As a result of SSL Legacy renegotiation error, Adding UnsafeLegacyRenegotiation option to the SSL conf file.

### CODE REVIEW

File: docker/python3/Dockerfile [hunk 1]
```diff
@@ -14,3 +14,16 @@ RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-
 
 RUN addgroup -g 4000 demisto \
   && adduser -u 4000 -G demisto -D demisto -s /bin/sh 
+
+# Handling the issue described here https://knowledgebase.paloaltonetworks.com/KCSArticleDetail?id=kA10g000000POJ0CAO&lang=en_US%E2%80%A9
+# by enabling UnsafeLegacyRenegotiation
+RUN echo 'openssl_conf = openssl_init\
```
Comment (JudahSchwartz) [inline docker/python3/Dockerfile:20]: can we have this in the folder as a file and COPY it in?

**Comment by JudahSchwartz** [comment]
reminder to add this to python3-deb and python3-native also

File: docker/python3/Dockerfile (hunk 1 above)
Comment (GuyAfik) [reply docker/python3/Dockerfile:20]: +1, there are other docker-images which should be updated, better to do in an external file and copy it

**Comment by GuyAfik** [comment]
> reminder to add this to python3-deb and python3-native powershel also

this should be done on **python3-ubi** and not on py3-native, py3-native "inherits" from python3-ubi

**Pull request closed without merging**